cql3/column_condition.cc

```diff
@@ -34,64 +34,55 @@ void validate_operation_on_durations(const abstract_type& type, cql3::expr::oper
     }
 }
 
-int is_satisfied_by(cql3::expr::oper_t op, const abstract_type& cell_type,
-        const abstract_type& param_type, const data_value& cell_value, const bytes& param) {
+} // end of anonymous namespace
 
-        std::strong_ordering rc = std::strong_ordering::equal;
-        // For multi-cell sets and lists, cell value is represented as a map,
-        // thanks to collections_as_maps flag in partition_slice. param, however,
-        // is represented as a set or list type.
-        // We must implement an own compare of two different representations
-        // to compare the two.
-        if (cell_type.is_map() && cell_type.is_multi_cell() && param_type.is_listlike()) {
-            const listlike_collection_type_impl& list_type = static_cast<const listlike_collection_type_impl&>(param_type);
-            const map_type_impl& map_type = static_cast<const map_type_impl&>(cell_type);
-            assert(list_type.is_multi_cell());
-            // Inverse comparison result since the order of arguments is inverse.
-            rc = 0 <=> list_type.compare_with_map(map_type, param, map_type.decompose(cell_value));
-        } else {
-            rc = cell_type.compare(cell_type.decompose(cell_value), param);
-        }
-        switch (op) {
-            using cql3::expr::oper_t;
-        case oper_t::EQ:
-            return rc == 0;
-        case oper_t::NEQ:
-            return rc != 0;
-        case oper_t::GTE:
-            return rc >= 0;
-        case oper_t::LTE:
-            return rc <= 0;
-        case oper_t::GT:
-            return rc > 0;
-        case oper_t::LT:
-            return rc < 0;
-        default:
-            assert(false);
-            return false;
-        }
-}
+namespace cql3 {
```

Comment:
Here too the mention of EQ_SPECIAL_NULL in the commit message is no longer correct.

Reply:
Will fix.